### PR TITLE
update Epson alpha release

### PR DESCRIPTION
**Pull request by aedler-ft**

### CODE REVIEW

File: product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
-# Epson Interoperabilität (in Entwicklung)
+# Epson Interoperabilität
 
 ## SCU
 
 Das _fiskaltrust.Middleware.SCU.DE.Epson_-Package ermöglicht die Verbindung der Middleware mit einer Epson-TSE.
 
-**Verfügbar ab Version**: 1.3.x (in Entwicklung)
+**Alpha Version verfügbar ab Version**: 1.3.10
```
Comment (StefanKert) [inline product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md]: Release candidate 1.3.10-rc1

**Review by MarkusEGruber — APPROVED** [review]

File: product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md [hunk 2]
```diff
@@ -15,6 +17,14 @@ Das _fiskaltrust.Middleware.SCU.DE.Epson_-Package ermöglicht die Verbindung der
 | _deviceid_ | Die für die TSE spezifische Geräte-ID. Falls nicht angegeben, wird der Defaultwert verwendet. | ja |
 | _timeout_ | Timeout-Wert für die Kommunikation mit der TSE in Millisekunden. Falls nicht angegeben, wird der Default-Wert von 60 Sekunden verwendet. | ja |
 
+## Konfiguration
+
+![scu-de-epson-config](../media/scu-de-epson-config.png)
+
+- **Host:** The DNS or IP Address of the Epson Server, Epson Printer or local webservice
```
Comment (StefanKert) [inline product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md]: If the rest of the article is in german we should also use german here :)

File: product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md [hunk 3]
```diff
@@ -15,6 +17,14 @@ Das _fiskaltrust.Middleware.SCU.DE.Epson_-Package ermöglicht die Verbindung der
 | _deviceid_ | Die für die TSE spezifische Geräte-ID. Falls nicht angegeben, wird der Defaultwert verwendet. | ja |
```
Comment (StefanKert) [inline product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md:17]: Don´t know if this section is still necessary?

File: product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md [hunk 4]
```diff
@@ -28,20 +38,10 @@ Für die Finalisierung der Epson TSE Imlementierung suchen wir Partner für eine
...
-
-**Artikel Nummer:** 4445-20580
-
-Technische Sicherungseinrichtung (TSE-Modul), Bauform: MicroSD-Karte, Lebensdauer: 20 Mio. Signaturen, Zertifikatslaufzeit: 3 Jahre
-
-Sonderartikel: Storno und Rückgabe unabhängig von der Lieferzeit ausgeschlossen.
-
-
-
 ### Epson TSE, USB
```
Comment (StefanKert) [inline product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md:45]: Should we mention the Printer and TSE Server here

File: product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md (hunk 2 above)
Comment (StefanKert) [reply product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md]: Actually we could use the old parameters and just change tseurl and tseport

File: product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md (hunk 4 above)
Comment (aedler-ft) [reply product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md:45]: agree, but later

File: product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md (hunk 2 above)
Comment (aedler-ft) [reply product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md]: jepp

File: product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md (hunk 1 above)
Comment (aedler-ft) [reply product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md]: done

File: product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md (hunk 3 above)
Comment (aedler-ft) [reply product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md:17]: i also don't know, there i'll leave it in for now

**Pull request merged**